packages/material-ui/src/Select/SelectInput.js

```diff
@@ -247,9 +247,10 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
   }
 
   return (
-    <div className={classes.root}>
+    <React.Fragment>
```

Comment:
React has a Short Hand for this
instead of `<React.Fragment>` can't we use `<>` ? Or is it that you all prefer `<React.Fragment>` than `<>` ?

Reply:
We prefer `React.Fragment`. In that form we can easily add props to the fragment (like `key`, or hopefully soon `ref`.

Reply:
You can't provide a key to the shorthand syntax. You won't find a single occurrence of it in the codebase.

Reply:
Okay got it.